tools/dispersion.go

```diff
@@ -689,9 +692,11 @@ func getDispersionReport(flags *flag.FlagSet) *dispersionReport {
 		}
 		report.ObjectReports = append(report.ObjectReports, objectReport)
 		detailRows, err := db.Query(`
-        SELECT create_date, partition, partition_item_path, items_found, items_need
-        FROM dispersion_report_detail 
-        WHERE rtype = "object" AND policy = ? ORDER BY create_date DESC LIMIT 1`, policy)
+			SELECT create_date, partition,
+			account || '/' || container || '/' object as partition_item_path,
```

Comment:
Similar comment as above: Should it be `'/' || object`?

Reply:
fixed